tests/pipelines/test_pipelines_common.py

```diff
@@ -227,7 +227,7 @@ class PipelineTesterMixin:
 
     # set these parameters to False in the child class if the pipeline does not support the corresponding functionality
     test_attention_slicing = True
-    test_cpu_offload = True
```

Comment:
Why are we discarding this?

Reply:
We always test it

Reply:
We completely remove the `test_cpu_offload` flag because every pipeline is now tested